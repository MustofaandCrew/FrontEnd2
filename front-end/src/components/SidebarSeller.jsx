import React from 'react';
import {
    CDBSidebar,
    CDBSidebarContent,
    CDBSidebarFooter,
    CDBSidebarHeader,
    CDBSidebarMenu,
    CDBSidebarMenuItem,
} from 'cdbreact';
import { NavLink } from 'react-router-dom';
import '../assets/css/SidebarSeller.css'

import Logo from '../assets/images/logo.png';

const SidebarSeller = () => {
    return (
        <div
            className='main'
        >
            <CDBSidebar className='sidebar'>
                <CDBSidebarHeader>
                    <a
                        href="/"
                        className="text-decoration-none"
                    >
                        <img src={Logo} alt="logo" />
                    </a>
                </CDBSidebarHeader>

                <CDBSidebarContent className="sidebar-content">
                    <CDBSidebarMenu>
                        <NavLink exact to="/transaksibuyer" activeClassName="activeClicked">
                            <CDBSidebarMenuItem icon="columns" className='text'>Dashboard</CDBSidebarMenuItem>
                        </NavLink>
                        <NavLink exact to="/accountbuyer" activeClassName="activeClicked">
                            <CDBSidebarMenuItem icon="table" className='text'>My Account</CDBSidebarMenuItem>
                        </NavLink>
                        <a exact to="/accountbuyer" activeClassName="activeClicked">
                            <CDBSidebarMenuItem className='mx-5'>qwertyuiopasdfghjklzxcvbnm</CDBSidebarMenuItem>
                        </a>
                    </CDBSidebarMenu>
                </CDBSidebarContent>

                <CDBSidebarFooter className="footer">
                    <div
                        className='footer-text'
                    >
                        Sidebar Footer
                    </div>
                </CDBSidebarFooter>
            </CDBSidebar>
        </div>
    );
};

export default SidebarSeller;