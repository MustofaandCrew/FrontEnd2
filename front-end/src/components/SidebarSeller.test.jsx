import React from "react";
import renderer from "react-test-renderer";
import SidebarSeller from "../components/SidebarSeller";


describe("Test suits for SidebarSeller", () => {
    it("should match with snapshot", () => {
        const tree = renderer.create(<SidebarSeller />).toJSON();
    });
});