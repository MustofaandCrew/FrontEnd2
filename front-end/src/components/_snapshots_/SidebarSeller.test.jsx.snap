// Jest Snapshot v1, https://goo.gl/fbAQLP
import SidebarSeller from "../components/SidebarSeller";

exports["SidebarSeller"] = {
    test: function test() {
        var tree = renderer.create(<SidebarSeller />).toJSON();
        expect(tree).toMatchSnapshot();
    }
};